### ISSUE TITLE
Add crypto economics section 

### ISSUE THREAD

**Issue by jlopp** [jlopp/lopp.net#139]
https://github.com/libbitcoin/libbitcoin/wiki/Cryptoeconomics

**Comment by jlopp** [comment]
Awesome list of resources https://github.com/jpantunes/awesome-cryptoeconomics

MIT Cryptoeconomics Lab Research https://ce.mit.edu/research/

**Comment by wordsarehelpful** [comment]
http://runbitcoin.org/ - might be more ELI5 but also has economics

**Comment by DARlucas** [comment]
Please consider adding this ELI15 (mildly technical) post on Bitcoin:

https://medium.com/digitalassetresearch/a-guide-to-bitcoins-technical-brilliance-for-non-programmers-e28211e797c0

**Comment by andhan360** [comment]
Ruben and others of Seoul bitcoin are doing a pretty good job:

https://m.youtube.com/channel/UC9ygnvBkGNTSNcfNjHAt6xQ/videos
